tools/accuracy_checker/accuracy_checker/adapters/audio_recognition.py

```diff
@@ -681,6 +681,28 @@ def empty(lm=None, start_with_bos=True):
     def logp_total(self):
         return log_sum_exp(self.logp_blank, self.logp_non_blank)
 
+class DumbDecoder(Adapter):
+    __provider__ = 'dumb_decoder'
+    prediction_types = (CharacterRecognitionPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'alphabet': ListField(
+                optional=True, default=None, value_type=str, allow_empty=False, description=
+                "Alphabet as list of strings. Default is space + 26 English letters + apostrophe."
+            ),
+        })
+        return parameters
+
+    def configure(self):
+        self.alphabet = self.get_value_from_config('alphabet') or ' ' + string.ascii_lowercase + '\''
+        self.alphabet = self.alphabet.encode('ascii').decode('utf-8')
+
+    def process(self, raw, identifiers=None, frame_meta=None):
+        decoded = ''.join(self.alphabet[t] for t in raw[0])
+        return [CharacterRecognitionPrediction(identifiers[0], decoded.upper())]
```

Comment:
I think moving to uppercase or not should be adapter parameter.
Also need to check that you have only 1 identifier (gove assert if not), if model does not support batching

Reply:
Please add new adapter to docs

Reply:
Fixed